java-manta-client-unshaded/src/main/java/com/joyent/manta/util/MantaUtils.java

```diff
@@ -376,6 +379,27 @@ public static String lastItemInPath(final String path) {
         return lastPart.toString();
     }
 
+    /**
+     * Generates an allowed bucket name from the path provided.
```

Comment:
What is this method for?  The implementation looks like it uses random UUIDs, but the neither the name or description hint at a non-idempotent operation.

Reply:
This method is contained in `MantaUtils` which is part of the user consumable SDK library. No code used in testing should be placed in this library if at all possible. If you need to create test utility methods, they should go in `TestSuiteSetup` or `IntegrationTestHelper`.

Reply:
Method defined in `IntegrationTestHelper`